projects/1.16/assets/apotheosis/apotheosis/lang/zh_cn.json

```
@@ -1,497 +1,581 @@
+	"enchantment.apotheosis.bane_of_illagers": "灾厄村民杀手",
+    "enchantment.apotheosis.bane_of_illagers.desc": "对灾厄村民的攻击伤害增加。",
+    "death.attack.apoth_corrupted": "%1$s 被腐化附魔榨干了。",
+    "death.attack.apoth_corrupted.player": "%s 被 %s 腐化了。",
+    "item.apotheosis.prismatic_web": "棱柱形蜘蛛网",
+    "info.apotheosis.prismatic_cobweb": "可以用来消除一件装备中的所有腐化附魔。",
+    "item.apotheosis.null_book": "其他宝典",
+    "item.apotheosis.armor_head_book": "头盔宝典",
+    "item.apotheosis.armor_chest_book": "胸甲宝典",
+    "item.apotheosis.armor_legs_book": "护腿宝典",
+    "item.apotheosis.armor_feet_book": "鞋子宝典",
+    "item.apotheosis.weapon_book": "武器宝典",
+    "item.apotheosis.digger_book": "矿工宝典",
+    "item.apotheosis.fishing_rod_book": "渔夫宝典",
+    "item.apotheosis.bow_book": "弓宝典",
+    "item.apotheosis.scrap_tome": "分解宝典",
+    "info.apotheosis.null_book": "只接受其他类型的附魔。",
+    "info.apotheosis.armor_head_book": "只接受头盔类型的附魔。",
+    "info.apotheosis.armor_chest_book": "只接受胸甲类型的附魔。",
+    "info.apotheosis.armor_legs_book": "只接受护腿类型的附魔。",
+    "info.apotheosis.armor_feet_book": "只接受鞋子类型的附魔。",
+    "info.apotheosis.weapon_book": "只接受武器类型的附魔。",
+    "info.apotheosis.digger_book": "只接受挖掘类型的附魔。",
+    "info.apotheosis.fishing_rod_book": "只接受垂钓类型的附魔。",
+    "info.apotheosis.bow_book": "只接受弓类型的附魔。",
+    "info.apotheosis.scrap_tome": "可以将一些附魔转移到分解宝典上，之后使用方法与附魔书相同。",
+    "info.apotheosis.scrap_tome2": "使用后将会被消耗。",
+    "info.apotheosis.table": "最大普通书架等级：%s",
+    "info.apotheosis.table2": "最大地狱书架等级：%s",
+    "enchantment.apotheosis.tempting": "引诱",
+    "enchantment.apotheosis.tempting.desc": "引诱附近的动物。",
+    "enchantment.apotheosis.shield_bash": "盾击",
+    "enchantment.apotheosis.shield_bash.desc": "增加盾牌造成的伤害。",
+    "enchantment.apotheosis.reflective": "反甲",
+    "enchantment.apotheosis.reflective.desc": "格挡时受到攻击有可能会对攻击者造成伤害。",
+    "enchantment.apotheosis.berserk": "狂战士",
+    "enchantment.apotheosis.berserk.desc": "受到伤害时会变得愤怒。",
+    "enchantment.apotheosis.knowledge": "阅历",
+    "enchantment.apotheosis.knowledge.desc": "敌人的掉落物将会被转化为经验。",
+    "enchantment.apotheosis.splitting": "拆分",
+    "enchantment.apotheosis.splitting.desc": "当拥有该附魔的铁砧砸中有多种附魔的附魔书时附魔书将会被拆分。",	
+	"enchantment.apotheosis.obliteration": "分裂",
+    "enchantment.apotheosis.obliteration.desc": "当拥有该附魔的铁砧砸中有单一附魔且等级高于 1 级的附魔书时附魔书将被拆分为 2 个，附魔等级下降 1 级。",
+    "enchantment.apotheosis.natures_blessing": "大自然的祝福",
+    "enchantment.apotheosis.natures_blessing.desc": "手持锄头右键可以直接对作物施加骨粉。",
+    "enchantment.apotheosis.rebounding": "反弹",
+    "enchantment.apotheosis.rebounding.desc": "被近战攻击时攻击者有可能会被弹飞。",
+    "enchantment.apotheosis.magic_protection": "魔法抗性",
+    "enchantment.apotheosis.magic_protection.desc": "减少玩家造成的魔法伤害，同时减少玩家受到的魔法伤害。",
+    "enchantment.apotheosis.crescendo": "快速射击",
+    "enchantment.apotheosis.crescendo.desc": "增加你的弩在装填完箭后瞄准时的射击次数。",
+    "block.apotheosis.prismatic_altar": "海之祭坛",
+    "comment_id": "/原版调整",
+	"generic.reachDistance": "攻击距离",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
	"generic.reachDistance": "触及距离",
```
非常严重的问题，会严重影响玩家的游玩。